docusaurus.config.js

```diff
@@ -142,16 +142,16 @@ module.exports = {
   trailingSlash: true,
   themeConfig: {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
-    announcementBar: {
-      id: "summit",
-      content: renderAnnouncementBar(
-        "Pulsar Virtual Summit Europe 2023",
-        "https://pulsar-summit.org/event/europe-2023"
-      ),
-      backgroundColor: "#282826",
-      textColor: "#fff",
-      isCloseable: false,
-    },
+    // announcementBar: {
```

Comment:
thought - perhaps we can develop a default announcementBar later.

Reply:
If you have a concrete suggestion of how it should work, I can implement it.

Reply:
I don't know, lol.

Other projects may set a banner for "star me", but I don't think it's a good idea.